apps/dsp-app/src/app/workspace/search/advanced-search/resource-and-property-selection/search-select-property/specify-property-value/search-date-value/search-date-value.component.ts

```diff
@@ -1,12 +1,10 @@
 import { Component, Inject, Input, OnDestroy, OnInit } from '@angular/core';
-import {
-    UntypedFormBuilder,
+import {UntypedFormBuilder,
```

Comment:
What is here the code convention for multiple imports?

Reply:
Basically whatever the formatter chooses when you right click > format document